### DISCUSSION TITLE
Target material by name?

### DISCUSSION THREAD

**Discussion by danilokrulj** [google/model-viewer#4032] [Q&A]
Hi everyone,

Title pretty much explains the question, is it possible to target materials by name? I will be switching between multiple models but will name the materials the same so i can manipulate them easier. If its not possible to target them by name does anyone have other suggestions?

Thanks in advance, and I wish everyone a happy New Year! :)

**Comment by elalish** [comment]
Not our best documentation, but yes, you can use `mv.model.getMaterialByName('name')`. You can find that and the rest of the material API here: https://modelviewer.dev/examples/scenegraph/#pickMaterialExample
